go/chat/boxer.go

```diff
@@ -749,6 +749,7 @@ func (b *Boxer) unversionHeaderMBV2(ctx context.Context, headerVersioned chat1.H
 			OutboxID:          hp.OutboxID,
 			OutboxInfo:        hp.OutboxInfo,
 			KbfsCryptKeysUsed: hp.KbfsCryptKeysUsed,
+			EphemeralMetadata: hp.EphemeralMetadata, // TODO only for v3
```

Comment:
this is a temporary measure to get tests running but should be changed in https://github.com/keybase/client/pull/11404/files